tajo-plan/src/main/java/org/apache/tajo/plan/rewrite/rules/CommonConditionReduceRule.java

```diff
@@ -162,58 +162,72 @@ protected EvalNode visitDefaultFunctionEval(Object context, Stack<EvalNode> stac
     private EvalNode rewrite(BinaryEval evalNode) {
       // Example qual: ( a | b ) ^ ( a | c )
```

Comment:
Even through it is not change in this patch, it would be great if you change it to more intuitive example.